packages/ove-asset-manager/README.md

```diff
@@ -1,7 +1,112 @@
-To run this Service you must add Object store credentials into the Appsettings.json file as follows
+# OVE Asset Manager Services
 
-  "s3Client": {
-    "AccessKey": "secret",
+This collection of microservices provides end users with a simple way of managing and processing content to be displayed using OVE. 
+
+## Asset Manager
+
+The core service is the **Asset Manager**. This enables media and content to be uploaded to an **S3** compatible **object store** and records metadata about its processing in an **SQL database**. A full user friendly website is provided to interact with the Asset Manager. A full RESTful API is provided for advanced users.  
+
+## Asset Services

```

Comment:
The next subsection refers to 'Asset Processing Microservices': it might be helpful to always include the 'Processing' (i.e. refer to 'Asset Processing Services' throughout).

Reply:
fixed